src/test/java/com/google/javascript/clutz/partial/es6_module.js

```diff
@@ -0,0 +1,9 @@
+import * as rel from './rel/module/name';
+import * as rel2 from '/abs/module/name';
+import * as abs from 'abs_strip_for_testing/module/name';
```

Comment:
Maybe doc that 'abs_strip_for_testing' is special-cased in the test harness?

Reply:
done